packages/widget-chat/src/utils/browser.js

```diff
@@ -31,8 +31,8 @@ export function isNotificationSupported() {
   return window && window.Notification;
 }
 
-export function requestPermissionForNotifications() {
-  return window.Notification.requestPermission();
+export function requestPermissionForNotifications(callback) {
```

Comment:
It was simply for having all of the Notification items in one utility vs having some functionality in the utility and some using straight `window.Notification`.